(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> <Rows (3)> is not JSON serializable"
p3
sS'layer'
p4
S'C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py'
p5
sS'code'
p6
S'# -*- coding: utf-8 -*-\n# probeer zoiets als\nfrom plistlib import Data\n\nimport matplotlib.pyplot as plt\nimport random\nimport plotly.plotly as py\nfrom plotly.graph_objs import *\n#py.tools.set_credentials_file(username=\'epicfacexp\', api_key=\'2j3arysr66\')\n\n\ndef index():\n    rows = db(db.sensor.id > 0).select()\n    return dict(message="hello from overzicht.py", rijen=rows)\n\n\ndef detail():\n    print("detail() in overzicht.py")\n    sensorNaam = request.vars[\'naam\']\n    time = db(db.data.sensor == sensorNaam).select(db.data.tijd)\n    temp = db(db.data.sensor == sensorNaam).select(db.data.temperatuur)\n\n    trace0 = Scatter(\n        x=time,\n        y=temp\n    )\n    data = Data([trace0])\n\n    url = py.plot(data, filename=\'basic-line\')\n\n    return dict(dataset="url")\n\nresponse._vars=response._caller(detail)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\gluon\\restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py", line 33, in <module>\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\gluon\\globals.py", line 417, in <lambda>\n    self._caller = lambda f: f()\n  File "C:\\Users\\Pieter-Jan\\PycharmProjects\\untitled\\web2py\\applications\\Sensors\\controllers/overzicht.py", line 29, in detail\n    url = py.plot(data, filename=\'basic-line\')\n  File "C:\\Python27\\lib\\site-packages\\plotly\\plotly\\plotly.py", line 241, in plot\n    res = _send_to_plotly(figure, **plot_options)\n  File "C:\\Python27\\lib\\site-packages\\plotly\\plotly\\plotly.py", line 1401, in _send_to_plotly\n    cls=utils.PlotlyJSONEncoder)\n  File "C:\\Python27\\lib\\json\\__init__.py", line 251, in dumps\n    sort_keys=sort_keys, **kw).encode(obj)\n  File "C:\\Python27\\lib\\site-packages\\plotly\\utils.py", line 145, in encode\n    encoded_o = super(PlotlyJSONEncoder, self).encode(o)\n  File "C:\\Python27\\lib\\json\\encoder.py", line 207, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n  File "C:\\Python27\\lib\\json\\encoder.py", line 270, in iterencode\n    return _iterencode(o, 0)\n  File "C:\\Python27\\lib\\site-packages\\plotly\\utils.py", line 211, in default\n    return json.JSONEncoder.default(self, obj)\n  File "C:\\Python27\\lib\\json\\encoder.py", line 184, in default\n    raise TypeError(repr(o) + " is not JSON serializable")\nTypeError: <Rows (3)> is not JSON serializable\n'
p11
s.